oak-commons/src/test/java/org/apache/jackrabbit/oak/commons/FileIOUtilsTest.java

```diff
@@ -116,7 +116,7 @@ public void writeCustomReadOrgStringsDeprecated() throws Exception {
         Set<String> actual = newHashSet("a", "z", "e", "b");
 
         File f = folder.newFile();
-        int count = writeStrings(added.iterator(), f, false, new Function<String, String>() {
+        int count = writeStrings(added.iterator(), f, false, new com.google.common.base.Function<String, String>() {
```

Comment:
Shouldn't we use the `Function` from the new jar or is this intentional? I see we are importing `org.apache.jackrabbit.oak.guava.common.base.Function` as well

Reply:
Sama as above.